packages/babel-core/src/transformation/file/options/build-config-chain.js

```diff
@@ -199,6 +206,14 @@ class ConfigChainBuilder {
 
     const envKey = babel.getEnv();
     if (options.env) {
+      // TODO: Remove in Babel v8
+      if (!warnedDeprecatedEnv) {
+        console.log(`The "env" option has been deprecated and will be removed in a future version of Babel.
```

Comment:
Could we use https://github.com/babel/babel/blob/97cd51995bfd999c82a4fb2da933668683bd43d2/packages/babel-core/src/transformation/file/logger.js#L41 there?